DISTRIBUTING.md

```diff
@@ -169,3 +169,352 @@ binary packages. It also includes miscellaneous tools to do things such as
 fetching the last good commit that passed the
 [Travis](https://travis-ci.org/JuliaLang/julia/builds) tests.
 
+
+# Point releasing 101
+
+Creating a point/patch release consists of several distinct steps.
+
+## Backporting commits
+
+Some pull requests are labeled "backport pending x.y", e.g. "backport pending 0.6".
+This designates that the next subsequent release tagged from the release-x.y branch
+should include the commit(s) in that pull request.
+Once the pull request is merged into master, each of the commits should be [cherry
+picked](https://git-scm.com/docs/git-cherry-pick) to a dedicated branch that will
+ultimately be merged into release-x.y.
+
+### Creating a backports branch
+
+First, create a new branch based on release-x.y.
+The typical convention for Julia branches is to prefix the branch name with your
+initials if it's intended to be a personal branch.
+For the sake of example, we'll say that the author of the branch is Jane Smith.
+
+```
+git fetch origin
+git checkout release-x.y
+git rebase origin/release-x.y
+git checkout -b js/backport-x.y
+```
+
+This ensures that your local copy of release-x.y is up to date with origin before
+you create a new branch from it.
+
+### Cherry picking commits
+
+Now we do the actual backporting.
+Find all merged pull requests labeled "backport pending x.y" in the GitHub web UI.
+For each of these, scroll to the bottom where it says "someperson merged commit
+`123ABC` into `master` XX minutes ago".
+Note that the commit name is a link; if you click it, you'll be shown the contents
+of the commit.
+If this page shows that `123ABC` is a merge commit, go back to the PR page---we
+don't want merge commits, we want the actual commits.
+However, if this does not show a merge commit, it means that the PR was squash-merged.
+In that case, use the git SHA of the commit, listed next to commit on this page.
+
+TODO
+
+## Checking for performance regressions
+
+Point releases should never introduce performance regressions.
+Luckily the Julia benchmarking bot, Nanosoldier, can run benchmarks against any
+branch, not just master.
+In this case we want to check the benchmark results of js/backport-x.y against
+release-x.y.
+To do this, we awaken the Nanosoldier from his robotic slumber using a comment
+on your backporting pull request:
```

Comment:
There seems to be an inconsistency here: either it should read "we ... our" or "you ... your". Or perhaps better:

> To do this, awaken the Nanosoldier from his robotic slumber using a comment on the backporting pull request